crates/backend/src/codegen.rs

```diff
@@ -1382,6 +1382,9 @@ impl<'a> ToTokens for DescribeImport<'a> {
 impl ToTokens for ast::Enum {
     fn to_tokens(&self, into: &mut TokenStream) {
         let enum_name = &self.rust_name;
+        let name_str = self.js_name.to_string();
+        let name_len = name_str.len() as u32;
+        let name_chars = name_str.chars().map(|c| c as u32).collect::<Vec<_>>();
```

Comment:
I believe `quote!` can handle iterators? If it does there would be no need to use `collect()`.